harness/setup.py

```diff
@@ -55,6 +55,7 @@
         "docker-compose>=1.13.0",
         "tqdm",
         "appdirs",
+        "backoff",
```

Comment:
I'm flagging this as requiring changes to task environments so we are not surprised on release-party day.

Reply:
I think this means that after this PR is merged and prior to the next release, I need to make sure that the task envs is rebuilt by adding any commit. Is that correct?

Reply:
Yes.